### DISCUSSION TITLE
Number of connection to Redis

### DISCUSSION THREAD

**Discussion by fmazzant** [ZiggyCreatures/FusionCache#346] [General]
How many connections are opened to Redis? Is it one per node, or can multiple connections be created for each individual node?
thank you so much.
Federico.

**Comment by jodydonetti** [comment]
Hi @fmazzant , in general you can have multiple connections to the same Redis instance from the same node, it's not a problem.

With FusionCache that depends on the `RedisCache` and `RedisBackplane` usage: you can have one per each or, to save some, share the same connection with a `RedisCache` and a `RedisBackplane` instances, thanks to the `ConnectionMultiplexerFactory` that you find in the `RedisCacheOptions` and `RedisBackplaneOptions`

I'm not sure this is what you wanted to know, hope this helps.
